scripts/test_json_schema.py

```diff
@@ -41,6 +41,14 @@ def get_version() -> str:
         "geometry": {
             "encoding": "WKB",
             "geometry_types": [],
+            "covering": {
+                "bbox": {
+                    "xmin": "bbox.xmin",
+                    "ymin": "bbox.ymin",
+                    "xmax": "bbox.xmax",
+                    "ymax": "bbox.ymax",
+                },
+            },
```

Comment:
I would leave it out here, because this is used just below to test the "minimal" required metadata (and covering is fully optional)

Reply:
Done. Just added it separately to the tests below